### PR TITLE
GitHub templates

### PR DESCRIPTION
**Pull request by arnau**
## Context

A part of open sourcing code is to be nice with the community. Github helps with 
https://github.com/openregister/managing-registers/community

## Changes proposed in this pull request

* Add an issue template.
* Add a pull request template.

Besides that, I would like someone more familiar with the development cycle to put together a `CONTRIBUTING.md` document. Either as part of this PR or in a new one.

/cc @openregister/product-team

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Steps to reproduce the expected behaviour
+
+
+### Expected behaviour
```
Comment (danmitchell-) [inline .github/ISSUE_TEMPLATE.md]: Would it be nice to have “actual behaviour” as well as “expected behaviour”?

Comment (fjavierm) [reply .github/ISSUE_TEMPLATE.md]: And maybe, some kind of 'Short description' at the beginning?

Comment (arnau) [reply .github/ISSUE_TEMPLATE.md]: Nice suggestions, thanks

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+### Overview
```
Comment (gidsg) [inline .github/ISSUE_TEMPLATE.md]: I prefer the following headings and order from the Mozilla bug template:

* What did you do? (steps to reproduce) 

* What happened? (actual results)

* What should have happened? (expected results)

Comment (arnau) [reply .github/ISSUE_TEMPLATE.md]: I'm easy, what the rest think about this?

Comment (gidsg) [reply .github/ISSUE_TEMPLATE.md]: Just to say mine is based on the Mozilla template described here: https://developer.mozilla.org/en-US/docs/Mozilla/QA/Bug_writing_guidelines#General_Outline_of_a_Bug_Report and usable here: https://bugzilla.mozilla.org/enter_bug.cgi#h=bugForm%7CFirefox (behind login)

**Comment by arnau** [comment]
@gidsg @fjavierm Given that no one else gave an opinion on the matter I changed the issue template with @gidsg suggestion. If you are happy with it I'll merge and we can revisit it later on.

**Comment by fjavierm** [comment]
@arnau Happy to go with it. I guess that this is one of the things that we will see if needs more specification when we/people start to use it. Right now, is very open, it depends a lot is the user filling the template add enough information on every section.

I prefer this approach that one with a lot of specific sections, it gives you more flexibility but, it gives you more responsibility.

**Pull request merged**